packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_service_runs.py

```diff
@@ -169,7 +169,7 @@ class ResourceTrackerServiceRunStatus(str, enum.Enum):
     ),
     # Run status
     sa.Column(
-        "service_run_status",
+        "service_run_status",  # Partial index was added manually in migration
```

Comment:
is there no way to achieve this in SQL Alchemy? this would create a discrepancy between the models and the migrations

Reply:
Done :+1: